common-lisp/parrot-refactoring-kata.asd

```diff
@@ -0,0 +1,22 @@
+;;;; parrot-refactoring-kata.asd
+
+(asdf:defsystem "parrot-refactoring-kata"
+  :description ""
+  :author "King Arthur king@roundtable.org"
```

Comment:
For this one you probably want to credit Monty Python (look up the dead parrot sketch)

Reply:
I am not very familiar with copyrights but wouldn't it be nicer to credit them in the readme of this kata?

Reply:
It's just a bit of fun! Most languages don't have a space in the build file to credit an author, so none of the others have this. It's really not important.

Reply:
Sorry, I just didn't get your intention. :-D
How do you like "Monty Python monty.python@legendary.fun"?

Reply:
Yes, sounds good!